colibre/auto_plotter/black_holes.yml

```
@@ -316,3 +316,141 @@ black_hole_mass_function_adaptive:
     - filename: BlackHoleMassFunction/Shankar2009_Data.hdf5
     - filename: BlackHoleMassFunction/Vika2009_Data.hdf5
     - filename: BlackHoleMassFunction/Caramete2010_Data.hdf5
+
+stellar_mass_black_hole_mass_30:
```

We already have a plot with the key name `stellar_mass_black_hole_mass_30` so we can't use it again.
```suggestion
stellar_mass_black_hole_mass_passive_30:
```